integration-test/test-pages/runtime-checks/config/replace-element.json

```diff
@@ -0,0 +1,21 @@
+{
+    "features": {
+        "runtimeChecks": {
+            "state": "enabled",
+            "exceptions": [],
+            "settings": {
+                "taintCheck": "enabled",
+                "matchAllDomains": "enabled",
+                "matchAllStackDomains": "enabled",
+                "overloadInstanceOf": "enabled",
+                "elementRemovalTimeout": 100,
+                "injectGlobalStyles": "enabled",
+                "replaceElement": "enabled",
```

Comment:
Should we add `monitorProperties` here as well? I realize probably not entirely necessary since it's default enabled, but for consistency.

Reply:
The other test relies on the default here too. I can clean this up after once we remove the algo.